webui/src/services/okrservice.js

```diff
@@ -44,12 +43,38 @@ export default class OkrService {
                 return dataHandler([]);
             }
 
-            let nodes = Array.from(document.querySelectorAll('li[data-id]'));
-            let objectives = nodes.map((each) => {
+            // Reference HTML
+            // <ul id="ul:{d4a13ad7-37ee-46d4-acdb-80b67c839aaa}{167}" data-id="objectives">
+            //     <li id="li:{82859f00-f0c9-4726-8174-2c0bd865a59b}{82}" data-id="llfsd39mx">
+            //         <p id="p:{82859f00-f0c9-4726-8174-2c0bd865a59b}{83}" style="margin-top:0pt;margin-bottom:0pt">Objective One</p>
+            //         <ul id="ul:{82859f00-f0c9-4726-8174-2c0bd865a59b}{84}">
+            //             <li id="li:{82859f00-f0c9-4726-8174-2c0bd865a59b}{88}">Key Result One</li>
+            //             <li id="li:{82859f00-f0c9-4726-8174-2c0bd865a59b}{92}">Key Result Two</li>
+            //         </ul>
+            //     </li>
+            //     <li id="li:{2db45fc0-8282-4d11-823f-011d2f7c8625}{33}" data-id="llfsd39mx">Objective Two</li>
+            // </ul>
```

Comment:
Wow! What is this?

Reply:
This OneNote's page structure. It is convenient to have it right here, where we are trying to parse it